.github/workflows/conda-package.yml

```diff
@@ -305,17 +311,27 @@ jobs:
       - name: Smoke test, step 1
         shell: cmd /C CALL {0}
         run: >-
-          conda activate dpctl_test && python -c "import sys; print(sys.executable)"
+          conda activate ${{ env.TEST_ENV_NAME }} && python -c "import sys; print(sys.executable)"
       - name: Smoke test, step 2
         shell: cmd /C CALL {0}
         run: >-
-          conda activate dpctl_test && python -m dpctl -f
+          conda activate ${{ env.TEST_ENV_NAME }} && python -m dpctl -f
+      - name: Create empty temporary directory to run tests from
+        shell: cmd /C CALL {0}
+        # create temporary empty folder to runs tests from
+        # https://github.com/pytest-dev/pytest/issues/11904
+        run: >-
+          mkdir "${{ env.workdir }}\test_tmp"
+      - name: List content of workdir folder
+        shell: cmd /C CALL {0}
+        run: dir "${{ env.workdir }}"
       - name: Run tests
         shell: cmd /C CALL {0}
         env:
           SYCL_CACHE_PERSISTENT: 1
+        working-directory: ${{ env.workdir }}\test_tmp
```

Comment:
I would allow default path, since it should also in writable location (how would `conda install` else work)? So conda environment location is located in writable path. Putting conda env in working folder could result in accidental removal and unexpected CI behavior. Same for other job.

Reply:
Good point, but this newly created folder is writable too. It is created by the workflow at the prior step as a sub-folder of the default path.